cmd/commands/cli/command.go

```diff
@@ -159,20 +168,112 @@ func (c *cliApp) handleActions(line string) {
 	}
 }
 
+func (c *cliApp) service(argsString string) {
+	printHelp := func() {
+		fmt.Println("service <action> [args]")
+		fmt.Println("	start	<ProviderID> <ServiceType> [options]")
+		fmt.Println("	stop	<ServiceID>")
+		fmt.Println("	list")
+		fmt.Println("	status	<ServiceID>")
+		fmt.Println("")
+		fmt.Println("example: service start 0x7d5ee3557775aed0b85d691b036769c17349db23 openvpn --openvpn.port=1194 --openvpn.proto=UDP")
+	}
+
+	args := strings.Fields(argsString)
+	if len(args) == 0 {
+		printHelp()
+		return
+	}
+
+	action := args[0]
+	switch action {
+	case "start":
+		if len(args) < 3 {
+			printHelp()
+			return
+		}
+		c.serviceStart(args[1], args[2], args[3:]...)
+	case "stop":
+		if len(args) < 2 {
+			printHelp()
+			return
+		}
+		c.serviceStop(args[1])
+	case "status":
+		if len(args) < 2 {
+			printHelp()
+			return
+		}
+		c.serviceGet(args[1])
+	case "list":
+		c.serviceList()
+	default:
+		info(fmt.Sprintf("Unknown action provided: %s", action))
+		printHelp()
+	}
+}
+
+func (c *cliApp) serviceStart(providerID, serviceType string, args ...string) {
+	opts, err := parseServiceOptions(serviceType, args...)
+	if err != nil {
+		info("Failed to parse service options:", err)
+		return
+	}
+
+	service, err := c.tequilapi.ServiceStart(providerID, serviceType, opts)
+	if err != nil {
+		info("Failed to start service: ", err)
+		return
+	}
+
+	status(service.Status, "ID: "+service.ID, "ProviderID: "+service.Proposal.ProviderID, "ServiceType: "+service.Proposal.ServiceType)
```

Comment:
can we reformat this a bit? IMHO it would look a bit cleaner this way

```golang
status(service.Status, 
fmt.Sprintf("ID: %v",service.ID),
fmt.Sprintf("ProviderID: %v",service.Proposal.ProviderID).
..
```

Reply:
Changed all of them.